.travis.yml

```diff
@@ -10,10 +10,20 @@ install: true
 matrix:
   include:
   - stage: test
+    name: "Test on Linux"
+    os: linux
     install:
     - "go get -u github.com/golang/dep/cmd/dep"
     - "go get -u github.com/vektra/mockery/.../"
     script: make build test verify-docs
+  - stage: test
+    name: "Test on Windows (Git Bash)"
+    os: windows
+    install:
+    - "choco install make"
+    - "go get -u github.com/golang/dep/cmd/dep"
+    - "go get -u github.com/vektra/mockery/.../"
+    script: make build test verify-docs
```

Comment:
I thought there was a problem with deleting files on windows+choco?

Reply:
@ericbottard Yes, there was a problem with trying to delete files using the Windows `rmdir` command but investigating that led to the realisation that Travis builds in Windows environments actually run in the Git Bash shell and the Makefile was actually invoking the `rmdir` GNU utility. Travis' choice of Git Bash means our Makefile works as-is (and hence the removal of all the WIP changes to it).

Reply:
Excellent!